gwcs/spectroscopy.py

```diff
@@ -67,11 +68,15 @@ class WavelengthFromGratingEquation(Model):
     r""" Solve the Grating Dispersion Law for the wavelength.
 
     .. Note:: This form of the equation can be used for paraxial
-    (small angle approximation) as well as oblique incident angles.
-    With paraxial systems the inputs are sin of the angles and it
-    transforms to :math:`\sin(alpha_in) + \sin(alpha_out) / m * d` .
-    With oblique angles the inputs are the direction cosines of the
-    angles.
+      (small angle approximation) as well as oblique incident angles.
+      With paraxial systems the inputs are ``sin`` of the angles and it
+      transforms to
+
+      :math:`(\sin(alpha_in) + \sin(alpha_out)) / (m * d)`.
```

Comment:
In isolation the use of m and d in the equation above do not appear to be defined.